app/tornado_handlers/download.py

```diff
@@ -14,44 +13,54 @@
 from pyulog.ulog2kml import convert_ulog2kml
 
 # this is needed for the following imports
-sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), '../plot_app'))
-from helper import get_log_filename, validate_log_id, \
-    flight_modes_table, load_ulog_file, get_default_parameters
+sys.path.append(
+    os.path.join(os.path.dirname(os.path.realpath(__file__)), "../plot_app")
+)
+from helper import (
+    get_log_filename,
+    validate_log_id,
+    flight_modes_table,
+    load_ulog_file,
+    get_default_parameters,
+)
```

Comment:
Could you please avoid unrelated style changes (or minimally separate by commits. In this case there's CI to check the style)? It makes things hard to review.